composer.json

```diff
@@ -5,7 +5,8 @@
     "license": "MIT",
     "require": {
         "php": ">=7.4",
-        "spryker-eco/unzer": "^1.0.0",
+        "spryker-eco/unzer": "dev-feature/cc-16934/dev-unzer-glue-support-integration",
```

Comment:
* Don't forget to change it to the "^1.1.0".

Reply:
Yep, will be done before release. Thanks for short reminder here.

Reply:
Fixed.